### PR TITLE
Add initAxis2ClientOptions and axis2xml attributes to Call mediator blocking mode

### PR DESCRIPTION
**Pull request by jananithangavel**

### CODE REVIEW

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp [hunk 1]
```diff
@@ -79,7 +79,25 @@
         callMediator.setEndpoint(ep);
     }
 
-    callMediator.setBlocking(Boolean.parseBoolean(request.getParameter("mediator.call.blocking")));
+    //Sets blocking information
+    String param=request.getParameter("mediator.call.blocking");
```
Comment (chanikag) [inline components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp]: Formatting issue: there should be a space before and after =

**Comment by sdkottegoda** [comment]
Does this have a dependancy on fixes in synapse? If so, can we please link those?

**Comment by jananithangavel** [comment]
This is associated with this PR https://github.com/wso2/wso2-synapse/pull/835, and should not be merged unless that PR in synapse is merged.

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp (hunk 1 above)
Comment (jananithangavel) [reply components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp]: Fixed

**Pull request merged**